src/v2/view-builder/views/EnrollProfileUpdateView.js

```diff
@@ -51,5 +51,19 @@ const Footer = BaseFooter.extend({
 
 export default BaseView.extend({
   Body,
-  Footer
+  Footer,
+
+  postRender() {
+    BaseView.prototype.postRender.apply(this, arguments);
+    const uiSchema = this.options.currentViewState.uiSchema;
+    /**
+     * As per requirement of this flow set secondEmail default to empty string
+     * ideally server should have passed default string in remediation
+     */
+    uiSchema.forEach(input => {
+      if (input.name === 'userProfile.secondEmail') {
+        this.model.set('userProfile.secondEmail', '');
+      }
+    });
```

Comment:
```suggestion
    const hasSecondEmail = uiSchema.find(({ name })=> name === 'userProfile.secondEmail');
    if (hasSecondEmail) this.model.set('userProfile.secondEmail', '');
```

Reply:
I am writing a prototype func `getSchemaByName`.. so that it can be used by any views.